workos/user_management.py

```diff
@@ -948,9 +989,9 @@ def authenticate_with_refresh_token(
             RefreshTokenAuthenticationResponse: Refresh Token Authentication response from WorkOS.
         """
 
-        payload = {
-            "client_id": workos.client_id,
-            "client_secret": workos.api_key,
+        payload: AuthenticateWithRefreshTokenParameters = {
+            "client_id": cast(str, workos.client_id),
+            "client_secret": cast(str, workos.api_key),
```

Comment:
These `cast`s might get fixed as part of allowing more than one set of credentials in the SDK

Reply:
Yeah, this whole method of plucking these variables from the workos.py feels wrong.

Reply:
Not feels. Is. It reduces flexibility using the SDK. While not required, I'd love to refactor to make it possible to create a fresh instance of the SDK with different credentials. I actually think we need figure out our approach to doing this soon, in case it changes how a developer would access the default client.